### PR TITLE
fixes for google authentication flow

### PR DESCRIPTION
**Pull request by michaelburch**
This allows for multiple responses with a 'code' parameter as seen in recent Google authentication flows and updates the default user agent string

### CODE REVIEW

File: src/LoginView.js [hunk 1]
```diff
@@ -64,7 +64,7 @@ export default class LoginView extends PureComponent {
     const result = adService.getLoginFlowResult(navState.url);
     if (
       result.requestType === RequestType.Ignore ||
-      result.requestType === RequestType.Code ||
+      result.requestType === RequestType.Code && !navState.loading ||
```
Comment (GSingh01) [inline src/LoginView.js:67]: please add unit test for this too

File: src/LoginView.js [hunk 2]
```diff
@@ -129,7 +129,7 @@ export default class LoginView extends PureComponent {
 
     return (
       <WebView
-        userAgent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10.13; rv:61.0) Gecko/20100101 Firefox/61.0"
+        userAgent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/13.1.2 Safari/605.1.15"
```
Comment (GSingh01) [inline src/LoginView.js:132]: Please verify if unit tests still pass, specially the snapshot

File: src/LoginView.js (hunk 1 above)
Comment (michaelburch) [reply src/LoginView.js:67]: unit test added

File: src/LoginView.js (hunk 2 above)
Comment (michaelburch) [reply src/LoginView.js:132]: updated snapshot, confirmed all tests pass

**Review by GSingh01 — APPROVED** [review]

**Pull request merged**